src/plugins/janus_streaming.c

```diff
@@ -2997,6 +3021,9 @@ json_t *janus_streaming_query_session(janus_plugin_session *handle) {
 					json_object_set_new(pd, "max-delay", json_integer(s->max_delay));
 					json_object_set_new(info, "playout-delay", pd);
 				}
+				if(session->abscapturetime_src_ext_id > 0) {
+					json_object_set_new(info, "abs-capture-time-src-ext-id", json_integer(session->abscapturetime_src_ext_id));
+				}
```

Comment:
Code style nit: no need for `{ .. }` brackets here, since the action is simple (single line).

Reply:
Fixed